src/babel/transformation/file/options.json

```diff
@@ -25,6 +25,11 @@
     "type": "string"
   },
 
+  "getModuleName": {
+    "description": "function to transform the default module name",
+    "type": "func"
```

Comment:
Remove the `type` property and add `"hidden": true` to stop it from appearing in the CLI options.